cyclist/src/edu/utexas/cycic/OutPut.java

```diff
@@ -127,12 +127,26 @@ public static void controlSetup(Document doc, Element control){
 		control.appendChild(simDuration);
 
 		Element simStartMon = doc.createElement("startmonth");
-		simStartMon.appendChild(doc.createTextNode(CycicScenarios.workingCycicScenario.simulationData.startMonth));
+		simStartMon.appendChild(doc.createTextNode(CycicScenarios.workingCycicScenario.simulationData.startMonth+1));
```

Comment:
is `startMonth` a string here? it looks like it's just concatenating the 1, not adding it.